packages/@okta/vuepress-theme-prose/assets/css/components/_multiTenancy.scss

```diff
@@ -0,0 +1,152 @@
+$blue: #163648;
+$darkBlue: #153b56;
+$darkerBlue: #142430;
+$gray: #d3d2d6;
+$darkGray: #20313b;
+$magenta: #e22866;
```

Comment:
When we do need to include CSS, we should should avoid hard-coding values like hex colors. We should use globally available tokens from Odyssey: https://odyssey.okta.design/base/color/

(Dev docs currently imports an old alpha version of Odyssey, so not everything documented in Odyssey is currently available here. But we do have an available `cv` function, such as used by [assets/css/components/_onThisPage.scss](https://github.com/okta/okta-developer-docs/blob/master/packages/%40okta/vuepress-theme-prose/assets/css/components/_onThisPage.scss), as well as an older color map defined in [assets/css/okta/base/variables.scss](https://github.com/okta/okta-developer-docs/blob/master/packages/%40okta/vuepress-theme-prose/assets/css/okta/base/variables.scss).)

Reply:
This is super helpful.  I'm aware of the cv function. Thanks for the links.